.github/workflows/triage.yml

```diff
@@ -20,42 +19,105 @@ jobs:
       - name: Retrieve the content of all columns on the board
```

Comment:
This description is probably outdated now? I think it now selects only field definitions?